doc/ledger3.texi

```diff
@@ -4635,13 +4635,16 @@ Ledger will include @samp{; transid: 767718} in the first transaction
 from the file above.
 
 @findex --invert
+@findex --auto-match
 @findex --account @var{STR}
 @findex --rich-data
 
-The @command{convert} command accepts three options. They are
-@option{--invert} which inverts the amount field, @option{--account
-@var{STR}} which you can use to specify the account to balance against,
-and @option{--rich-data} which stores additional metadata as tags.
+The @command{convert} command accepts four options. They are
```

Comment:
As I am French, I usually follow the typography rule that order me to write numbers in letters. I searched the Internet if there is something similar for English typography. Actually there is a rule, less strict, that recommend to spell numbers inferior to ten. See http://www.dailywritingtips.com/10-rules-for-writing-numbers-and-numerals/ and http://everything2.com/title/rules+of+typography